presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestHiveBucketedTables.java

```diff
@@ -197,6 +199,20 @@ public void testSelectFromEmptyBucketedTableEmptyTablesAllowed()
                 .containsExactly(row(0));
     }
 
+    @Test(groups = {HIVE_CONNECTOR})
+    public void testIgnorePartitionBucketingIfNotBucketed()
+            throws SQLException
+    {
+        String tableName = mutableTableInstanceOf(BUCKETED_PARTITIONED_NATION).getNameInDatabase();
+        populateDataToHiveTable(tableName, NATION.getName(), Optional.of("part_key = 'insert_1'"));
+        populateDataToHiveTable(tableName, NATION.getName(), Optional.of("part_key = 'insert_2'"));
+
+        testContext().getDependency(QueryExecutor.class, "hive").executeQuery(format("ALTER TABLE %s NOT CLUSTERED", tableName));
```

Comment:
does it matter if you insert data after the alter table?

Reply:
Yes, because you want to be able to query data that was inserted when the table was both bucketed and partitioned. The use case that https://github.com/Teradata/presto/pull/710/commits/35caaee33bb0f543d6f2019d5a1a3b762fa5fd3d covers is when the data was inserted when the table was bucketed and partitioned and later on the user removed the bucketing.

Reply:
ok.  just checking that you didn't also need to insert data afterwards.

Reply:
Ya, it's all good, thanks for the review!